packages/xchain-ethereum/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # v.x.x.x
 
+### Update
```

Comment:
^ Breaking change

Pls prepare v.0.6.0 to publish it after merging this PR into master, thx